go.mod

```diff
@@ -83,3 +82,5 @@ require (
 	k8s.io/client-go v12.0.0+incompatible
 	k8s.io/klog v0.3.3
 )
+
+go 1.13
```

Comment:
Can we remove this? If you checkout docker images in fission-bundle, builder, fetcher we are using golang:1.12-alpine which has GOLANG_VERSION=1.12.17

Reply:
Done.